gulpfile.js

```diff
@@ -218,6 +218,15 @@ gulp.task('copyLocalEnvConfig', function () {
     .pipe(gulp.dest('config/env'));
 });
 
+// Make sure upload directory exists
+gulp.task('makeUploadsDir', function () {
+  return fs.mkdir('modules/users/client/img/profile/uploads', function (res) {
```

Comment:
Where is this convention defined?

This is a callback method for success/error and is not specifically a callback only for errors.